server/app/src/androidTest/java/sh/calaba/instrumentationbackend/actions/device/UiautomatorScrollToElement.java

```diff
@@ -0,0 +1,82 @@
+package sh.calaba.instrumentationbackend.actions.device;
+
+import androidx.test.uiautomator.Configurator;
+import androidx.test.uiautomator.UiObjectNotFoundException;
+import androidx.test.uiautomator.UiScrollableCustom;
+import androidx.test.uiautomator.UiSelector;
+import sh.calaba.instrumentationbackend.InstrumentationBackend;
+import sh.calaba.instrumentationbackend.Result;
+import sh.calaba.instrumentationbackend.actions.Action;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+import static sh.calaba.instrumentationbackend.actions.device.StrategyUtils.convertBySelectorStrategyToUiSelectorStrategy;
+import static sh.calaba.instrumentationbackend.actions.device.StrategyVerifier.verifyStrategy;
+
+public class UiautomatorScrollToElement implements Action {
+    @Override
+    public Result execute(String... args) {
+        InstrumentationBackend.getUiDevice();
+        Configurator.getInstance().setScrollAcknowledgmentTimeout(2000l);
```

Comment:
After each scrolling method(ex: scrollForward) it will wait for the specified time. This delay will help to load the content. I will add this comment explanation.